wpilib/wpilib/spi.py

```diff
@@ -236,7 +236,7 @@ def read(self, initiate: bool, size: int) -> bytes:
         else:
             return hal.readSPI(self.port, size)
 
-    def transaction(self, dataToSend: bytes) -> bytes:
+    def transaction(self, dataToSend: bytes) -> Union[bytes, List[int]]:
```

Comment:
uhhhh we should probably change the HAL to return bytes here

Reply:
I totally changed the wrong thing. I meant to change the parameter, not return value